wgsl/index.bs

```diff
@@ -488,12 +491,11 @@ The following is a list of keywords which are reserved for future expansion.
   <tr><td>`NOT_EQUAL`<td>!==
   <tr><td>`GREATER_THAN`<td>>
   <tr><td>`GREATER_THAN_EQUAL`<td>>=
-  <tr><td>`LOGICAL_SHIFT_RIGHT`<td>>>
-  <tr><td>`ARITH_SHIFT_RIGHT`<td>>>>
+  <tr><td>`SHIFT_RIGHT`<td>>>
   <tr><td>`LESS_THAN`<td><
   <tr><td>`LESS_THAN_EQUAL`<td><=
   <tr><td>`SHIFT_LEFT`<td><<
```

Comment:
nit: These should really (eventually) be escaped somehow to avoid weird html parsing issues. I'd just put them in code tags: `<<` etc.